src/js/constants/HealthBar.js

```diff
@@ -0,0 +1,20 @@
+const HealthBarMap = {
+  tasksUnknown: {
+    cssName: 'unknown',
+    label: 'Unknown'
+  },
+  tasksHealthy: {
+    cssName: 'healthy',
+    label: 'Healthy'
+  },
+  tasksUnhealthy: {
+    cssName: 'unhealthy',
+    label: 'Unhealthy'
+  },
+  tasksStaged: {
+    cssName: 'staged',
```

Comment:
Could you rename `cssName` to `className`. It's more consistent with the rest of the codebase.